### DISCUSSION TITLE
How to differentiate a storage backend per tenant?

### DISCUSSION THREAD

**Discussion by husnialhamdani** [cortexproject/cortex#5713] [Q&A]
What is the approach on differentiating a storage per tenant using Cortex?

**Comment by yeya24** [comment]
Hi @husnialhamdani, may I know your usecase of this? You wanna specify a different storage path per tenant or different storage backends per tenant?

**Reply by husnialhamdani** [reply]
Different storage backends per tenant, as we want to guarantee the tenant data is completely isolated.

**Reply by husnialhamdani** [reply]
Do you have any suggestion for this @yeya24?
